generated/rflx-rflx_scalar_sequence.adb

```diff
@@ -12,16 +14,16 @@ is
       Last_Bit : Types.Bit_Index;
       First    : Types.Index;
       Last     : Types.Index;
-      Offset   : Types.Offset;
+      Offst    : Types.Offset;
```

Comment:
Why was this renaming required? IMHO `Offst` is not a very good abbreviation.

Reply:
This is needed because of hiding checks and https://github.com/Componolit/Workarounds/issues/24. As `Types` is use-ed the type `Offset` will be hidden by the variable `Offset`. I'm open for suggestions for a better abbreviation.

Reply:
If this is due to a tool issue, I'd propose to leave the parameter (as ugly) as is and add a comment pointing to https://github.com/Componolit/Workarounds/issues/24 so we remember to change it back to `Offset` later...

Reply:
What kept me from adding a comment is that this comment will be in the generated code. I think such a comment should just be a reminder for developers and should not be seen by a users of the tool.

Reply:
You're right. A respective comment in the generator should be sufficient.

Reply:
One option would be to filter out comments when generating files from templates.

Reply:
I missed the point that this is in a template. Maybe we should filter out only comments that have a special syntax or tag in it?

Reply:
Fixed by 9121c1d.